pyproject.toml

```diff
@@ -1,28 +1,35 @@
-[tool.poetry]
-name = "aiod"
-version = "0.1.0"
+[project]
+name = "aiondemand"
+version = "0.0.1"
 description = "Python SDK for the AIoD metadata catalogue"
 authors = [
-    "Jean Matias <smatias.jean@gmail.com>",
-    "Pieter Gijsbers <p.gijsbers@tue.nl>",
+    { name="Jean Matias", email= "smatias.jean@gmail.com"},
+    { name="Pieter Gijsbers", email="p.gijsbers@tue.nl"},
 ]
 license = "MIT"
 readme = "docs/README.md"
 
-[tool.poetry.dependencies]
-python = ">=3.11"
-requests = "^2.31.0"
-pandas = "^2.2.1"
-aiohttp = "^3.9.4"
-asyncio = "^3.4.3"
+requires-python = ">=3.11"
 
-[tool.poetry.group.dev.dependencies]
-pre-commit = "^3.5.0"
-responses = "^0.25.0"
-aioresponses = "^0.7.6"
-pytest = "^8.1.1"
-pytest-asyncio = "^0.23.6"
+dependencies = [
+  "requests",
+  "pandas",
+  "aiohttp",
+]
+
+[project.optional-dependencies]
+dev = [
+    "pre-commit",
+    "responses",
+    "aioresponses",
+    "pytest",
+    "pytest-asyncio",
+]
+
+[tool.commitizen]
+name = "cz_conventional_commits"
+version = "1.0.2"
+#update_changelog_on_bump = true
+version_files = ["pyproject.toml:version", "src/aiod/__version__.py"]
+#changelog_start_rev = "1.0.0"
```

Comment:
Do we need these comments here?

Reply:
I forgot to remove them. Probably want to add this back later. In the original workflow the project uses conventional commits and the change log is updated automatically for each release based on those.